packages/jbrowse-cli/README.md

```diff
@@ -25,7 +25,7 @@ $ npm install -g @gmod/jbrowse-cli
 $ jbrowse COMMAND
 running command...
 $ jbrowse (-v|--version|version)
-@gmod/jbrowse-cli/0.0.0 linux-x64 node-v12.14.1
+@gmod/jbrowse-cli/0.0.0 darwin-x64 node-v14.0.0
```

Comment:
this keeps getting changed by whoever runs the program. we might want to consider not outputting system information

Reply:
yeah, this section probably won't really change, so we can make it static instead of auto-generated (just take out the `<!-- usage -->` and `<!-- usagestop -->`)

Reply:
Righto